ablator/main/state.py

```diff
@@ -314,6 +585,23 @@ def __append_trial(
         optuna_trial_num: int,
         trial_state: TrialState,
     ) -> bool:
+        """
+        Append a trial to the database.
```

Comment:
There's another optuna database, and this only update the experiment database, so maybe specify this to avoid confusion